reset.css

```diff
@@ -0,0 +1,48 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
```

Comment:
rather than using so many selectors here, I'd try using a global selector. You can override the global settings on specific elements if you select them after you set your global style.